dist/ng-material-multilevel-menu/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ng-material-multilevel-menu",
-  "version": "3.0.2",
+  "name": "sb-material-multilevel-menu",
```

Comment:
I think the name should be `ng-material-multilevel-menu`

Reply:
corrected